File: paciente/src/main/java/com/sevenine/conecta/repository/entity/ParentescoEntity.java
package com.sevenine.conecta.repository.entity;

import lombok.Getter;
import lombok.Setter;

import javax.persistence.Entity;
import javax.persistence.Id;
import javax.persistence.Table;

@Getter
@Setter
@Table(name = "parentesco")
@Entity
public class ParentescoEntity {

    @Id
    private Long id;

    private String descricao;

}

File: service-conecta/src/main/java/com/sevenine/conecta/adapter/output/entity/ParentescoEntity.kt
package com.sevenine.conecta.adapter.output.entity;

import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.Table;

@Table(name = "parentesco")
@Entity(name = "Parentesco")
data class ParentescoEntity(
        @Id
        val id: Long,
        val descricao: String,
)
